protocol/messenger.go

```diff
@@ -4595,19 +4628,26 @@ func (m *Messenger) saveDataAndPrepareResponse(messageState *ReceivedMessageStat
 
 	for _, message := range messageState.Response.messages {
 		if _, ok := newMessagesIds[message.ID]; ok {
+
 			message.New = true
 
+			if message.ContentType == protobuf.ChatMessage_IMAGE {
```

Comment:
Just a reminder to add/update tests.

Reply:
Completely forgot, Thanks for the reminder!

Reply:
Tried writing tests, But I faced a problem that the message content type when sent to a community has a type of `protobuf.ChatMessage_COMMUNITY` and doesn't allow it to be converted as a message as I presume, Because calling `GetImage` on it returns nil even though it has a message.

Reply:
mmh, pretty sure you can write a test for it, you don't even have to use a community for testing this feature, you can test it with a one to one chat

Reply:
Tested mentioning a user with images on a 1-on-1 chat, And only got an unviewed-messages indicator on the home screen above the chat where the message happened.